README.adoc

```diff
@@ -88,6 +88,22 @@ assert_equal(true, actual)
 assert(actual)
 ----
 
+=== Refute falsey [[refute-falsey]]
+
+Use `refute` if expecting falsey value.
```

Comment:
IMHO, falsey value is `false` and `nil`. However `assert_nil` will be used if an expected value is `nil`.
So it may be expected to be limited to `false` instead of falsey value. What do you think?

Reply:
Makes total sense for me @koic

Reply:
@koic I did update the doc